src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -112,6 +112,11 @@ class GemUtils {
 
                 systemProperties 'file.encoding' : 'utf-8'
             }
+            project.javaexec {
```

Comment:
it is no problem to run ```gem install ....``` from a ruby script which then could add the end run this ```generate_dir_info``` from jruby.

see https://github.com/jruby/jruby/blob/master/lib/pom.rb#L149